geode-dunit/src/main/java/org/apache/geode/cache/query/QueryTestUtils.java

```diff
@@ -1452,64 +1452,64 @@ public void initializeQueryMap() {
 
 
     // Queries used for Equi-join tests
-    queries.put("1199", "select * from /region1 r1, /region2 r2 where" + "r1.ID = r2.id");
```

Comment:
It seems like these must be unused. How did these queries even work previously? If they are unused is there a missing test or should these queries just be removed?

Reply:
Nope, we need to write tests to use them. Commons will have a discussion later this week to put a task in to use these queries or come up with an alternative. Right now, we are keeping these queries.